src/opnsense/mvc/app/models/OPNsense/Unbound/Unbound.xml

```diff
@@ -226,11 +226,19 @@
                 <ValidationMessage>A valid domain must be specified.</ValidationMessage>
                 <MaskPerItem>Y</MaskPerItem>
             </wildcards>
+            <rcode type="OptionField">
+                <Default>1</Default>
+                <Required>Y</Required>
+                <OptionValues>
+                    <opt1 value="REFUSED">REFUSED</opt1>
```

Comment:
```suggestion
                    <NOERROR>REFUSED</NOERROR>
```

let's keep this simple, the `value=` construct is usually implemented when values are not valid xml tags